tests/unit/requirements.txt

```diff
@@ -51,6 +51,9 @@ passlib[argon2]
 proxmoxer < 2.0.0 ; python_version >= '2.7' and python_version <= '3.6'
 proxmoxer ; python_version > '3.6'
 
+# requirements for the proxmox_pct_remote connection plugin
+paramiko >= 3.0.0 ; python_version >= '3.6'
```

Comment:
I belueve this is redundant since we only have integration tests, and this file is only used for unit testing.

Reply:
tests/unit/plugins/connection/test_proxmox_pct_remote.py

We got unit test 😉

Reply:
oops, missed that one